docs/practices/start.md

```diff
@@ -0,0 +1,127 @@
+# How to start developing nearcore
+
+## Running a local node
+
+If you're looking to run a validator or RPC node in mainnet or testnet, you can follow this [article](https://near-nodes.io/validator/running-a-node) that is using nearup or [this article](https://docs.near.org/pt-BR/docs/develop/node/validator/compile-and-run-a-node/) that shows you how to compile and run it.
+
+In this section, we'll cover the preferred setup that you might want to have when you do the local nearcore development.
+
+### Step 1 - get the code
+
+```shell
+git clone https://github.com/near/nearcore
+
+git checkout master
+```
```

Comment:
````suggestion
```console
$ git clone https://github.com/near/nearcore && cd nearcore
```
````

There's no need to checkout master, but cding into folder might help